pkg/scaffold/internal/templates/gopkg.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
This file should be removed.

Reply:
It is used in the update.

Reply:
See https://github.com/kubernetes-sigs/kubebuilder/pull/1355#discussion_r417417819.

Reply:
Removed . I agree that it shows no longer required at all.

Reply:
I agree that we could/should remove. After thinking about and checking it I just think that it should be better done in a follow-up PR. note that we need also change the docs and check all places that are speaking about it. So, shows out of the scope.